docs/developer.md

```diff
@@ -144,7 +141,7 @@ When you're done, you can run
 
 1. Create queue & Submit job
 ```bash
-go run ./cmd/armadactl/main.go create queue test --priorityFactor 1
+go run ./cmd/armadactl/main.go create queue --queueName test --priorityFactor 1
 go run ./cmd/armadactl/main.go submit ./example/jobs.yaml
```

Comment:
Good spot, but note this is broken elsewhere under docs/ too.

Reply:
ah yes- they should be fixed now